hedera-mirror-rest-java/src/main/java/com/hedera/mirror/restjava/mapper/NftAllowanceMapper.java

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright (C) 2019-2024 Hedera Hashgraph, LLC
```

Comment:
Please check all new files.

```suggestion
 * Copyright (C) 2024 Hedera Hashgraph, LLC
```